docker-compose.yaml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+services:
+   app:
+     build:
+       context:
+       dockerfile: dockerfile
+     ports:
+        - "8080:8080"
+     environment:
+        - SPRING_PROFILES_ACTIVE=docker   #please edit this as your current value.
+        - JAVA_OPTS=-Xmx512m -Xms256m
+     depends_on:
+        -db
+     volumes:
+        - ./build/reports:/app/reports
+   db:
+    image: postgres:13
+    ports:
+      - "5432:5432"
+    environment:      #configure this as yor db id passwd.
+      - POSTGRES_DB=wccdb
+      - POSTGRES_USER=wccuser
+      - POSTGRES_PASSWORD=wccpassword
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+sonarqube:
```

Comment:
Could you update README to use docker compose, please and also add a description in your PR.

Reply:
ok.